pkg/cmd/cli/cmd/importimage.go

```diff
@@ -347,6 +347,11 @@ func (o *ImportImageOptions) createImageImport() (*imageapi.ImageStream, *imagea
 				if len(from) == 0 {
 					from = stream.Spec.DockerImageRepository
 				}
+				// if the from is still empty this means there's no such tag defined
+				// nor we can't create any from .spec.dockerImageRepository
+				if len(from) == 0 {
+					return nil, nil, fmt.Errorf("the tag %q does not exists on the image stream - use the 'tag' command to create it or choose a different tag to import", tag)
```

Comment:
`exist` instead of `exists`.  For second part, `choose an existing tag to import or use the 'tag' command to create a new tag`